src/driver/test/SecondaryAllowed.yml

```diff
@@ -0,0 +1,51 @@
+SchemaVersion: 2018-07-01
+Actors:
+    - Name: StepdownCommand
+      Type: AdminCommand
+      Threads: 1
+      ExecutionStrategy:
+          Retries: 0
+          ThrowOnFailure: false
+      Phases:
+        - NoOp: true
+        - Phase: 1
+          Repeat: 1
+          Operation:
+              OperationMetricsName: "Stepdown"
```

Comment:
Can I convince you to test out OperationAwaitStepdown from [TIG-1322](https://github.com/mongodb/genny/commit/e24bca65dcc8a052c00d54fa0346cd898d75a560)?